.github/workflows/release.yml

```
@@ -0,0 +1,78 @@
+name: Mint release
```

```suggestion
name: Mint release 🍬
```
[optional] Just for fun!